### PR TITLE
about me

### PR DESCRIPTION
**Pull request by janeglanzer**

### CODE REVIEW

**Review by aadlander — CHANGES_REQUESTED** [review]
You don't have punctuation at the end of your statement.

File: about_me.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 print("My name is Andy and I    ")
 
-print("My name is Jane and I    ")
+print("My name is Jane and I am hungry   ")
```
Comment (mwcraig) [inline about_me.py]: How can you possibly be hungry with so much sugar sitting on the center table?

Comment (stefannelson) [inline about_me.py]: # Same.

Comment (stefannelson) [inline about_me.py]: # Same.

**Review by stefannelson — CHANGES_REQUESTED** [review]

File: about_me.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 print("My name is Andy and I    ")
 
-print("My name is Jane and I    ")
+print("My name is Jane and I will not add punctuation to the end of this sentence   ")
```
Comment (aadlander) [inline about_me.py:5]: sassy

**Review by aadlander — APPROVED** [review]

**Review by stefannelson — APPROVED** [review]

File: about_me.py (hunk 2 above)
Comment (stefannelson) [reply about_me.py:5]: # Wow

Comment (stefannelson) [inline about_me.py]: $f(x) = x^2$

**Pull request merged**